ddapm_test_agent/agent.py

```diff
@@ -1948,6 +1961,18 @@ def main(args: Optional[List[str]] = None) -> None:
         default=os.environ.get("VCR_CI_MODE", False),
         help="Will change the test agent to record VCR cassettes in CI mode, throwing an error if a cassette is not found on /vcr/{provider}",
     )
+    parser.add_argument(
+        "--web-ui-port",
+        type=int,
+        default=int(os.environ.get("WEB_UI_PORT", 0)),
+        help="Port to serve the optional web UI (default: disabled). Example: --web-ui-port=8080",
+    )
+    parser.add_argument(
```

Comment:
can we add docs to the README for both of these options? We should call out that the web ui one is experimental and not to be relied on